terraform/README.md

```diff
@@ -6,8 +6,13 @@ The input variables are documented in their description and it's best to refer
 to [variables.tf](variables.tf).
 
 ## Outputs
-- `unit` - the ignition systemd unit file
-- `file` - the ignition file to setup the ssh-key-agent binary
+Binary use:
+- `unit` - the ignition systemd unit file (binary)
+- `file` - ssh-key-agent binary ignition file
+- `download-unit` - systemd service to download the ssh-key-agent binary
```

Comment:
I would add a bit more context here, why use it and it being a substitution of `file`